tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
```

Comment:
The config can be simplified, there are some flags turned on for features not used in this PR (e.g., decorators, JSX).

Reply:
Done